fluidreview/api.py

```diff
@@ -165,7 +165,9 @@ def process_user(fluid_user):
     Args:
         fluid_user (ReturnDict): Data from a fluidreview.serializers.UserSerializer object
     """
-    user, _ = User.objects.get_or_create(email=fluid_user['email'], defaults={'username': fluid_user['email']})
+    user, _ = User.objects.get_or_create(email__iexact=fluid_user['email'],
+                                         defaults={'email': fluid_user['email'],
+                                                   'username': fluid_user['email']})
```

Comment:
Nit - I think this style is a little more readable:

```python
user, _ = User.objects.get_or_create(
    email__iexact=fluid_user['email'],
    defaults={'email': fluid_user['email'], 'username': fluid_user['email']}
)
```

You can take or leave that. Just a personal preference

Reply:
Thanks, updated it.